d1lod/d1lod/jobs.py

```diff
@@ -51,9 +51,9 @@
 QUEUE_MAX_SIZE_STANDOFF = 60  # (seconds) time to sleep before trying again
 
 # Set up connections to services
-SESAME_HOST = os.getenv('WEB_1_PORT_8080_TCP_ADDR', 'localhost')
-SESAME_PORT = os.getenv('WEB_1_PORT_8080_TCP_PORT', '8080')
-SESAME_REPOSITORY = 'd1lod'
+VIRTUOSO_HOST = "virtuoso"
```

Comment:
My original implementation used env vars which allowed some flexibility that I liked. Looking at the code, probably incorrectly. I don't remember why I wrote `WEB_1_PORT_8080_TCP_ADDR` which looks hacky and messy. Since Docker seamlessly manages the  `virtuoso` hostname for us, this change is much better!

It'd be nice, though, if this and the lines of code below it, failed over better for local development. If the env vars aren't set, I'd like this script to fall back to `localhost:8890` so that I can run individual components outside of Docker against a locally installed Virtuoso (or other SPARQL endpoint) if needed.

Since we're abstracting this to any SPARQL endpoint, it seems like HOST/PORT/REPOSITORY could be all combined into just an env var like "SPARQL_URI" with a value set in the `docker-compose.yml` like `SPARQL_URI=http://virtuoso:8890/sparql`.

What do you think?